src/xdocs/checks.xml

```diff
@@ -531,6 +531,13 @@
             <td>Checks for missing Javadoc comments for class, enum, interface, and annotation
             interface definitions.</td>
           </tr>
+          <tr>
+            <td><a href="config_javadoc.html#MissingLeadingAsterisk">MissingLeadingAsterisk</a></td>
+            <td>
+              Checks if the javadoc has <a href="https://docs.oracle.com/en/java/javase/14/docs/specs/javadoc/doc-comment-spec.html#leading-asterisks">leading asterisk</a>
```

Comment:
wrap long line
only URL content is allowed to be non-breakable and long.

Reply:
Done